lib/helpers/check-attributes.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const calculateLocationDisplay = require('./calculate-location-display');
+const supportsAttribute = require('./supports-attribute');
+
+module.exports = function checkAttributes(node, supported) {
+  let sourceReference = calculateLocationDisplay(null, node.loc.start);
```

Comment:
Passing `null` here as the `moduleName`, because I didn't find a way to access the template file name from within the AST transform. Any idea how this could work?

I am trying to give a useful hint to where the actual violation is located. Throwing an error will at least show the offending file downstream, but not the warning obviously...

Reply:
@rwjblue any idea about this?

Reply:
Done now as suggested.